src/applications/personalization/dashboard/components/CernerWidgets.jsx

```diff
@@ -170,28 +169,14 @@ CernerAlertBoxV2.propTypes = {
 };
 
 export const CernerWidget = ({ facilityLocations, authenticatedWithSSOe }) => (
-  <Toggler
-    toggleName={Toggler.TOGGLE_NAMES.vaOnlineSchedulingStaticLandingPage}
-  >
-    <div data-testid="cerner-widget">
-      <Toggler.Enabled>
-        <CernerAlertBoxV2
-          facilityLocations={facilityLocations}
-          primaryCtaButtonUrl={getCernerURL('')}
-          secondaryCtaButtonText="Go to My HealtheVet"
-          secondaryCtaButtonUrl={mhvUrl(authenticatedWithSSOe, 'home')}
-        />
-      </Toggler.Enabled>
-      <Toggler.Disabled>
-        <CernerAlertBox
```

Comment:
I'm assuming this component can be removed as well since it's not used anywhere else.

Reply:
Yup, will do.